Changelog.minor.md

```diff
@@ -15,6 +15,8 @@ release the new version.
 
 ### Changed
 
+-   #1101: Updated J-Link version to v8.18.
+-   #1101: Updated shared version to 207.0.0.
```

Comment:
But this PR doesn't update shared. :-) And even if it _did_ update shared, I would not mention it. We never do that. For J-Link it is something else, because that _is_ more significant.

Reply:
Woops, I've been updating shared on the side so I simply assumed so...
Fair either way, will remove this one line.